### DISCUSSION TITLE
Making laravel/sanctum tokens's identifiable

### DISCUSSION THREAD

**Discussion by marzvrover** [laravel/framework#47977] [Ideas]
Opening this discussion up to discuss a [draft PR](https://github.com/laravel/sanctum/pull/459) I've made to make tokens generated with Laravel Sanctum identifiable. I believe this increase the security postures of solutions using Laravel Sanctum by allowing for developers using minted tokens to quickly identify tokens in their code base using tools such as [GitHub's secret scanning](https://docs.github.com/en/code-security/secret-scanning/about-secret-scanning).

Disclosure: I work for GitHub and I am the main engineer involved with the secret scanning partner program.

**Comment by crynobone** [comment]
Seem like the PR has been merge and we can close this.
